packages/bonito-ui/src/environment/mock-browser-environment.ts

```diff
@@ -38,6 +38,24 @@ export class MockBrowserEnvironment
                 this.config.enableA11yTesting = true;
             }
         }
+
+        // some JSDOM APIs are not implemented, so we need to mock them
+        // to make importing monaco-editor works
+        // https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
+        Object.defineProperty(window, "matchMedia", {
+            writable: true,
+            value: jest.fn().mockImplementation((query) => ({
+                matches: false,
+                media: query,
+                onchange: null,
+                addListener: jest.fn(), // deprecated
```

Comment:
Let's move this into Jest-specific code. MockBrowserEnvironment can also be used for the playground, and doesn't require Jest. It also runs in a real browser, so doesn't need these stubs.

Reply:
moved jsdom mocks to setup-tests